lib/include/zstd.h

```diff
@@ -0,0 +1 @@
+../zstd.h
```

Comment:
?
This seems a strange content for a `*.h` header file ?

Reply:
Because SPM needs a separated `publicHeadersPath `, so I create a symbol link to `../zstd.h`

Reply:
What do you call a "symbol link" ?
Is that a specific SPM concept, or is it the standard unix' `ln -s` link ?

Reply:
Yes, It is a symbol link created by `ln -s`